docs/docs/wikitext103.md

```diff
@@ -0,0 +1,205 @@
+# WikiText-103
+
+![Banner](img/banner.png)
+[//]: # (TODO Change Me) 
```

Comment:
please fix - you can mirror the style of the other task docs closely (e.g. imagenet and coco)